test/d2l-scroll-wrapper.html

```diff
@@ -263,10 +195,6 @@
 				}, 1);
 			});
 
-			it('should apply --d2l-scroll-wrapper-left', function() {
```

Comment:
Hmm, how were these tests still passing?

Reply:
Oh nvm, we do not appear to be running any of these tests 🙃

Reply:
Haha yeah, commented out tests are the best tests.